owner-extras/src/main/java/org/aeonbits/owner/loaders/MongoPropertiesLoader.java

```diff
@@ -0,0 +1,77 @@
+package org.aeonbits.owner.loaders;
+
+import java.io.IOException;
+import java.net.URI;
+import java.util.Properties;
+
+import org.aeonbits.owner.loaders.Loader;
+import org.bson.Document;
+
+import com.mongodb.MongoClient;
+import com.mongodb.MongoClientURI;
+import com.mongodb.client.FindIterable;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoCursor;
+import com.mongodb.client.MongoDatabase;
+
+/**
```

Comment:
JavaDocs should be written as

/**
 * some text here.
 * some other text.
 */

Not this way. See JavaDocs in the rest of the project or refer to: http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html